twindb_backup/destination/base_destination.py

```diff
@@ -115,7 +119,13 @@ def status(self, status=None):
         :return: dictionary with the status
         """
         if status:
-            return self._write_status(status)
+            status['checksum'] = hashlib.md5(
```

Comment:
The method adds a key to the dictionary. We should better describe the status format.
Also, I see `status` type is not specified in the docstring - it should be.

Reply:
Checksum is only for internal use, not for public. I've noticed it in code + updated docstring.